keyboards/ericrlau/numdiscipline/info.json

```diff
@@ -0,0 +1,21 @@
+{
+    "keyboard_name": "ELau NumDiscipline KB", 
+    "url": "https://github.com/ericrlau/NumDiscipline", 
+    "maintainer": "ericrlau", 
+    "width": 19.5, 
+    "height": 5.25, 
+    "layouts": {
+        "LAYOUT_65": {
```

Comment:
```suggestion
        "LAYOUT_all": {
```

Reply:
yep, good call. I should have changed these all now that it isn't 65.